prime-router/src/main/kotlin/fhirengine/translation/hl7/FhirToHl7Converter.kt

```diff
@@ -152,7 +152,8 @@ class FhirToHl7Converter(
         var retVal = ""
         run findValue@{
             element.value.forEach {
-                val value = if (it.isBlank()) ""
+                val value = if(element.valueSet.isNotEmpty()) it
```

Comment:
I am not sure what this is doing.  Can you explain? `if(element.valueSet.isNotEmpty()) it`

Reply:
Reverted this as it was a bad way to solve an issue I had with a test